Library/Formula/otto.rb

```diff
@@ -0,0 +1,205 @@
+      :revision => "8102d0ed5ea2709ade1243798785888175f6e415"
+  end
+
+  go_resource "github.com/mitchellh/colorstring" do
+    url "https://github.com/mitchellh/colorstring.git",
+      :revision => "8631ce90f28644f54aeedcb3e389a85174e067d1"
+  end
+
+  go_resource "github.com/mitchellh/copystructure" do
+    url "https://github.com/mitchellh/copystructure.git",
+      :revision => "6fc66267e9da7d155a9d3bd489e00dad02666dc6"
+  end
+
+  go_resource "github.com/mitchellh/go-homedir" do
+    url "https://github.com/mitchellh/go-homedir.git",
+      :revision => "d682a8f0cf139663a984ff12528da460ca963de9"
+  end
+
+  go_resource "github.com/mitchellh/ioprogress" do
+    url "https://github.com/mitchellh/ioprogress.git",
+      :revision => "8163955264568045f462ae7e2d6d07b2001fc997"
+  end
+
+  go_resource "github.com/mitchellh/mapstructure" do
+    url "https://github.com/mitchellh/mapstructure.git",
+      :revision => "281073eb9eb092240d33ef253c404f1cca550309"
+  end
+
+  go_resource "github.com/mitchellh/osext" do
+    url "https://github.com/mitchellh/osext.git",
+      :revision => "5e2d6d41470f99c881826dedd8c526728b783c9c"
+  end
+
+  go_resource "github.com/mitchellh/panicwrap" do
+    url "https://github.com/mitchellh/panicwrap.git",
+      :revision => "1655d88c8ff7495ae9d2c19fd8f445f4657e22b0"
+  end
+
+  go_resource "github.com/mitchellh/prefixedio" do
+    url "https://github.com/mitchellh/prefixedio.git",
+      :revision => "89d9b535996bf0a185f85b59578f2e245f9e1724"
+  end
+
+  go_resource "github.com/mitchellh/reflectwalk" do
+    url "https://github.com/mitchellh/reflectwalk.git",
+      :revision => "eecf4c70c626c7cfbb95c90195bc34d386c74ac6"
+  end
+
+  go_resource "github.com/vaughan0/go-ini" do
+    url "https://github.com/vaughan0/go-ini.git",
+      :revision => "a98ad7ee00ec53921f08832bc06ecf7fd600e6a1"
+  end
+
+  go_resource "golang.org/x/crypto" do
+    url "https://go.googlesource.com/crypto.git",
+      :revision => "c8b9e6388ef638d5a8a9d865c634befdc46a6784"
+  end
+
+  go_resource "gopkg.in/flosch/pongo2.v3" do
+    url "https://gopkg.in/flosch/pongo2.v3.git",
+      :revision => "5e81b817a0c48c1c57cdf1a9056cf76bdee02ca9"
+  end
+
+  def install
+    contents = Dir["{*,.git,.gitignore}"]
+    gopath = buildpath/"gopath"
+    (gopath/"src/github.com/hashicorp/otto").install contents
+
+    ENV["GOPATH"] = gopath
+    ENV.prepend_create_path "PATH", gopath/"bin"
+
+    Language::Go.stage_deps resources, gopath/"src"
+
+    cd gopath/"src/github.com/jteeuwen/go-bindata/go-bindata" do
+      system "go", "install"
+    end
+
+    cd gopath/"src/golang.org/x/tools/cmd/stringer" do
+      system "go", "install"
+    end
+
+    cd gopath/"src/github.com/mitchellh/gox" do
+      system "go", "install"
+    end
+
+    cd gopath/"src/github.com/hashicorp/otto" do
+      system "make", "dev"
+      bin.install "bin/otto"
+    end
+  end
+
+  test do
+    system "#{bin}/otto", "--version"
```

Comment:
Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!